.golangci.yml

```diff
@@ -85,11 +85,29 @@ linters-settings:
 
           - cosmossdk.io/core
           - cosmossdk.io/errors
+          - cosmossdk.io/log
           - cosmossdk.io/math
           - cosmossdk.io/store
-          - github.com/cosmos/go-bip39
+          - cosmossdk.io/x/evidence
+          - cosmossdk.io/x/evidence/keeper
+          - cosmossdk.io/x/evidence/types
```

Comment:
You should be okay to just have the root package here for these, e.g. `cosmossdk.io/x/evidence`. Basically, whatever's in `go.mod`.

I'm not 100% sure on this though, but it seems to work just fine to have `github.com/cosmos/cosmos-sdk` to cover all the packages contained in that library.

Reply:
Actually. It's probably okay to leave it like this for now. Later, once the linter's fully passing, if we care, we can clean up some entries.